substrate/frame/src/lib.rs

```diff
@@ -203,8 +203,12 @@ pub mod prelude {
 	/// Dispatch types from `frame-support`, other fundamental traits
 	#[doc(no_inline)]
 	pub use frame_support::dispatch::{GetDispatchInfo, PostDispatchInfo};
-	pub use frame_support::traits::{
-		Contains, EstimateNextSessionRotation, IsSubType, OnRuntimeUpgrade, OneSessionHandler,
+	pub use frame_support::{
+		defensive, defensive_assert,
+		traits::{
+			tokens::ConvertRank, Contains, EitherOf, EstimateNextSessionRotation, IsSubType,
+			MapSuccess, NoOpPoll, OnRuntimeUpgrade, OneSessionHandler,
```

Comment:
I am not sure `ConvertRank` is really prelude but it can be ok.

Reply:
Our rule here from other reviews was that if a trait is used in 3 or more pallets, it can be in prelude, if not directly imported in the migration.